<?xml version="1.0"?>
<launch>
    <!-- Load the parameters for the SAC algorithm -->
    <rosparam command="load" file="$(find my_gurdy_description)/config/params_sac.yaml" />
    
    <!-- Launch the training node -->
    <node pkg="my_gurdy_description" type="train_gurdy_sac.py" name="gurdy_sac" output="screen"/>
</launch>